### DISCUSSION TITLE
也许我们可以删除`README_chinese.md`?

### DISCUSSION THREAD

**Discussion by lovelydinosaur** [encode/httpx#3024] [General]
我们只有一种语言的翻译似乎有点奇怪。

**Comment by zanieb** [comment]
It is the second most popular language, seems okay unless it's a maintenance issue?

**Reply by lovelydinosaur** [reply]
Es un problema de mantenimiento dado que no tenemos una buena manera de mantenerlo actualizado. Y parece un poco inconsistente.

**Reply by lovelydinosaur** [reply]
> unless it's a maintenance issue?

We're not keeping it up to date with `README.md`, so it is slightly a maintenance issue.

As much as I appreciate us reaching out to our international friends, it seems oddly balanced that we've got a canonical text with a single language translation? Would any non-native speakers like to give their perspective?

**Comment by Tunglies** [comment]
The README has been translated into Chinese, while the rest of the documentation remains in English. This creates a disjointed reading experience, but those who are interested in learning how to use httpx will likely browse the English documentation on their own.
只有 README 被翻译成中文，而文档仍是英文。阅读起来有些割裂感，实际上愿意学httpx使用的人会自行浏览英文文档。

I speak Chinese, which is my native language.

**Reply by lovelydinosaur** [reply]
Okay, thanks @Tunglies.

I've suggested the change, here... https://github.com/encode/httpx/pull/3122
